scene/gui/file_dialog.h

```diff
@@ -159,8 +159,10 @@ class FileDialog : public ConfirmationDialog {
 	PropertyListHelper property_helper;
 
 	inline static bool default_show_hidden_files = false;
+	static inline DisplayMode default_display_mode = DISPLAY_THUMBNAILS;
 	bool show_hidden_files = false;
 	bool use_native_dialog = false;
+	bool disable_overwrite_warning = false;
```

Comment:
I wonder if this could be a positive instead? As in `show_overwrite_warning`. Just a nitpick, not a fan of negative booleans.

Reply:
I could also turn it into customization flag.

Reply:
That sounds even better.